templates/etc/haproxy/haproxy-global.cfg.j2

```diff
@@ -78,4 +78,9 @@ global
 {% endif %}
 {% if haproxy_global_server_state_base is defined %}
     server-state-base {{ haproxy_global_server_state_base }}
+{% endif %}
+{% if haproxy_global_toggles is defined and haproxy_global_toggles|length %}
```

Comment:
Please add the the spaces between the `|`, if not `ansible-lint` will complain :laughing:

```
{% if haproxy_global_toggles is defined and haproxy_global_toggles|length %}
```
To:
```
{% if haproxy_global_toggles is defined and haproxy_global_toggles | length %}
```

Reply:
Hi! do I need to change it like this in whole file? It's fine by me, just don't want to do it twice :)

Reply:
> Hi! do I need to change it like this in whole file (see lines 72 or 67 for example)? It's fine by me, just don't want to do it twice :)

If you found some others yes please :+1:

Reply:
Alright, i think i got all of them now.